src/tests/infusion/getInfusion.test.js

```diff
@@ -1,5 +1,9 @@
 import supertest from 'supertest';
 import app from '../../http/app';
```

Comment:
I noticed the test runner already imports `supertest` and `app`. is there a reason for still importing them on this file. Am assuming its because of the `beforeAll hook` but still need confirmation

Reply:
Yes, it's because of the beforeAll hook. I am making an API call to create an infusion in the before all hook so I needed supertest and the app.